extensions/amp-story/1.0/page-advancement.js

```diff
@@ -577,6 +603,15 @@ class ManualAdvancement extends AdvancementConfig {
       return;
     }
 
+    if (this.isHandledByAffiliateLink_(target)) {
+      event.preventDefault();
```

Comment:
Is this enough to prevent the link analytics from triggering, or do we also need to call `stopPropagation`? Would be really awesome to have as a unit test as it's really at the core of this feature

Reply:
Does that mean adding `amp-analytics` to test, or are link analytics built in?

Reply:
Yes, you can test by adding `amp-analytics` and some configuration to start tracking click events on one specific link, eg:

```
<amp-analytics type="googleanalytics">
<script type="application/json">
{
  "vars": {
    "account": "UA-YYYY-Y"
  },
  "triggers": {
    "click on #foo trigger": {
      "on": "click",
      "selector": "#foo",
      "request": "event",
      "vars": {
        "eventCategory": "examples",
        "eventAction": "clicked-foo"
      }
    }
  }
}
</script>
</amp-analytics>
```

I've been testing and the issue we have right now is that one analytics event is sent when we expand the link, and one more when we actually navigate. I've been testing by looking at if this line of code is executed:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/events.js#L441

It looks like simply calling `event.stopPropagation();` here to prevent the event from bubbling up when we expand the link will fix the issue.